control/lib/gridmanager.js

```diff
@@ -109,6 +109,7 @@ wax.gm = function() {
 
         wax.request.get(gurl, function(err, t) {
             if (err) return callback(err, null);
+            if (t.errors_with_context) return callback(t.errors_with_context, null);
```

Comment:
Sorry for being a PITA. But having `errors_with_context` array we can check `type` and `subtype` properties for each error. IMHO, we can go for some error categorization. can't we?

Reply:
I thought this issue to make carto.js aware of every error on wax but I can change it.

Reply:
Oh! I just got it.

If Carto.js is responsible of categorizing them, it's perfect to yield error handling to Carto.js.

Sorry for the noise